ldms/src/sampler/zfs_common/Makefile.am

```diff
@@ -0,0 +1,14 @@
+noinst_LTLIBRARIES =
+dist_man1_MANS =
+
+AUTOMAKE_OPTIONS = subdir-objects
+
+AM_CPPFLAGS = @OVIS_INCLUDE_ABS@ \
```

Comment:
I would tend to prefer to just use libzfs_common_helo_la_CPPFLAGS rather than AM_CPPFLAGS here.

Reply:
done